dev-tools/startup/creator-node.sh

```diff
@@ -10,7 +10,7 @@ export delegatePrivateKey=$(printenv "CN${replica}_SP_OWNER_PRIVATE_KEY")
 export spOwnerWallet=$(printenv "CN${replica}_SP_OWNER_ADDRESS")
 export spID=$replica
 
-export creatorNodeEndpoint="http://$(nslookup "$(hostname -i)" | sed -n 's/.*name = \(.*\)/\1/p'):4000"
```

Comment:
Pls don't format this file

Reply:
this was part of the revert since every creator node has this standard name now and doesn't need port 4000 anymore

Reply:
I meant the formatting where you move the comments inline

Reply:
oh yea my bad I changed that back